tests/std/tests/P2538R1_adl_proof_std_projected/test.compile.pass.cpp

```diff
@@ -69,13 +91,22 @@ using simple_identity      = tagged_identity<void>;
 using simple_left_selector = tagged_left_selector<void>;
 using simple_zero_equality = tagged_zero_equality<void>;
 
-using validator                = holder<incomplete>*;
-using validating_truth         = tagged_truth<holder<incomplete>>;
-using validating_equal         = tagged_equal<holder<incomplete>>;
-using validating_less          = tagged_less<holder<incomplete>>;
-using validating_identity      = tagged_identity<holder<incomplete>>;
-using validating_left_selector = tagged_left_selector<holder<incomplete>>;
-using validating_zero_equality = tagged_zero_equality<holder<incomplete>>;
+using validator                  = holder<incomplete>*;
+using validating_truth           = tagged_truth<holder<incomplete>>;
+using validating_zeroer          = tagged_zeroer<holder<incomplete>>;
+using validating_true_comparator = tagged_true_comparator<holder<incomplete>>;
+using validating_equal           = tagged_equal<holder<incomplete>>;
+using validating_less            = tagged_less<holder<incomplete>>;
+using validating_identity        = tagged_identity<holder<incomplete>>;
+using validating_left_selector   = tagged_left_selector<holder<incomplete>>;
+using validating_zero_equality   = tagged_zero_equality<holder<incomplete>>;
+using validating_derived         = tagged_derived<holder<incomplete>>;
+using validating_base            = tagged_base<holder<incomplete>>;
+
+static_assert(std::sentinel_for<validating_base*, validating_derived*>);
+#ifndef __EDG__
+static_assert(!std::sized_sentinel_for<validating_base*, validating_derived*>);
+#endif // !defined(__EDG__)

```

Comment:
We should cite the bug number here, and use a "no workaround" comment now that #4362 has made us consistent.